index.html

```diff
@@ -66,20 +66,20 @@ <h1>Projects</h1>
 			<div class="sectionContent">
                 <article>
                     <h2>BikeRacks</h2>
-                    <p class="subDetails">Interactive map</p>
+                    <p class="subDetails">Platform for outsourcing bikerack location and quality</p>

```

Comment:
"outsourcing" has a specific meaning, especially in tech. This is crowdsourcing.